Makefile

```diff
@@ -89,7 +89,6 @@ release-tag:
 release-manifests:
 	mkdir -p _output
 	kubectl kustomize manifests/release > _output/components.yaml
-	echo "Please upload file _output/components.yaml to GitHub release"
```

Comment:
Can we update RELEASE.md file to mention how to interact with CI?

Reply:
I wonder if this is really meaningful since the assets will be automatically added to the release once it is published. There won't be anything for the release manager to do

Reply:
SG